src/core/Akka/Akka.csproj

```diff
@@ -253,19 +252,18 @@
     <Compile Include="IO\UdpSender.cs" />
     <Compile Include="IO\WithUdpSend.cs" />
     <Compile Include="Pattern\BackoffSupervisorBase.cs" />
+    <Compile Include="Pattern\BackoffSupervisor.cs" />
     <Compile Include="Pattern\CircuitBreaker.cs" />
     <Compile Include="Pattern\CircuitBreakerState.cs" />
     <Compile Include="Pattern\FutureTimeoutSupport.cs" />
     <Compile Include="Pattern\OpenCircuitException.cs" />
     <Compile Include="Routing\ConsistentHashRouter.cs" />
     <Compile Include="Serialization\JavaSerializer.cs" />
     <Compile Include="IO\ByteBuffer.cs" />
-    <Compile Include="IO\DirectByteBufferPool.cs" />
+    <Compile Include="IO\SocketEventArgsPool.cs" />
     <Compile Include="IO\Inet.cs" />
     <Compile Include="IO\IO.cs" />
-    <Compile Include="IO\SelectionHandler.cs" />
```

Comment:
Not me. Selection handlers were first thing, Willie removed while working on this branch. They were ported from the JDK NIO to provide an abstraction, that will make straightforward port of akka.io possible. But even on JVM people hate selection handlers design.

In .NET we don't have NIO, we use SocketAsyncEventArgs instead. This is one of the major changes on this branch.